python/cudf/cudf/core/udf/strings_typing.py

```diff
@@ -1,226 +1,273 @@
-# Copyright (c) 2022, NVIDIA CORPORATION.
+# Copyright (c) 2022-2023, NVIDIA CORPORATION.
 
 import operator
 
+import numpy as np
 from numba import types
+from numba.core.extending import models, register_model
 from numba.core.typing import signature as nb_signature
 from numba.core.typing.templates import AbstractTemplate, AttributeTemplate
 from numba.cuda.cudadecl import registry as cuda_decl_registry
 
-from strings_udf._typing import (
-    StringView,
-    bool_binary_funcs,
-    id_unary_funcs,
-    int_binary_funcs,
-    size_type,
-    string_return_attrs,
-    string_unary_funcs,
-    string_view,
-    udf_string,
-)
+import rmm
 
-from cudf.core.udf import masked_typing
-from cudf.core.udf._ops import comparison_ops
-from cudf.core.udf.masked_typing import MaskedType
+# libcudf size_type
+size_type = types.int32
```

Comment:
Let's move this definition next to where we hardcode other libcudf information like `size_type_dtype` (or reuse that declaration, if possible). https://github.com/rapidsai/cudf/blob/6a67e8fe49661f7f2f6d8da119996f683a21803a/python/cudf/cudf/_lib/types.pyx#L20